kpatch-build/kpatch-elf.c

```diff
@@ -186,6 +186,7 @@ void kpatch_create_rela_list(struct kpatch_elf *kelf, struct section *sec)
 		rela->type = GELF_R_TYPE(rela->rela.r_info);
 		rela->addend = rela->rela.r_addend;
 		rela->offset = rela->rela.r_offset;
+		rela->need_dynrela = 0;
```

Comment:
No need for this explicit initialization, we already memset the rela struct to zero in ALLOC_LINK.